### PR TITLE
Feature tg 47 add/edit project page tincho

### PR DESCRIPTION
**Pull request by martinlotufo93**

### CODE REVIEW

File: src/Components/Project/index.js [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+    try {
+      const data = await fetch(`${process.env.REACT_APP_API_URL}/employees/`);
+      const employees = await data.json();
+      const newEmployees = employees.data.map((employee) => {
+        return {
+          label: `${employee.firstName} ${employee.lastName}`,
+          value: employee._id
+        };
+      });
+      console.log('mensaje prueba', newEmployees);
```
Comment (HernanCazeneuve) [inline src/Components/Project/index.js:69]: Remember to delete checkpoint console log

File: src/Components/Project/index.js [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+      })
+    };
+    if (projectId) {
+      options.method = 'PUT';
+      url = `${process.env.REACT_APP_API_URL}/projects/${projectId}`;
+    }
+    try {
+      const response = await fetch(url, options);
+      const data = await response.json();
+      console.log(data);
```
Comment (HernanCazeneuve) [inline src/Components/Project/index.js:105]: Could also delete this console log

**Review by HernanCazeneuve — CHANGES_REQUESTED** [review]

**Review by AlejandroSantini — DISMISSED** [review]
Good job!

**Review by HernanCazeneuve — DISMISSED** [review]

**Review by HiginiaMed — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -8,6 +8,7 @@
     "@testing-library/user-event": "^12.8.3",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-dropdown": "^1.10.0",
```
Comment (franco-marini) [inline package.json:11]: Remove this dependency we should not add new libreries

**Review by franco-marini — CHANGES_REQUESTED** [review]

**Review by Gearyandres94 — APPROVED** [review]

File: src/Components/Project/index.js [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+      const params = new URLSearchParams(window.location.search);
+      const projectId = params.get('id');
+      if (projectId) {
+        const data = await fetch(`${process.env.REACT_APP_API_URL}/projects/${projectId}`);
+        const Projectfetch = await data.json();
+        let startDate = Projectfetch.data.startDate.slice(0, 10);
+        let endDate = Projectfetch.data.endDate.slice(0, 10);
+        setnameValue(Projectfetch.data.name);
+        setstatusValue(Projectfetch.data.status);
+        setdescriptionValue(Projectfetch.data.description);
```
Comment (franco-marini) [inline src/Components/Project/index.js:70]: The capitalized letter at first should be only for components or classes. Please fix this also

**Review by franco-marini — APPROVED** [review]

**Review by Gearyandres94 — APPROVED** [review]
Great job!!!

**Review by HiginiaMed — APPROVED** [review]

**Pull request merged**